fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -1569,7 +1574,10 @@ public void handleRunningTablets() {
 
         // 2. release ctx
         timeoutTablets.stream().forEach(t -> {
-            releaseTabletCtx(t, TabletSchedCtx.State.CANCELLED);
+            // Set "resetReplicaState" to true because
+            // the timeout task should also be considered as UNRECOVERABLE,
+            // so need to reset replica state.
+            releaseTabletCtx(t, TabletSchedCtx.State.CANCELLED, true);
```

Comment:
It is better to add a log here, it is useful for us to find why resetReplicaState is called. This log will be not too many.

Reply:
No need, this is an origin logic, and we don't expect this log in the past.